### ISSUE TITLE
Option for alternative non-blur effect

### ISSUE THREAD

**Issue by emcniece** [akveo/blur-admin#22]
I get that the blur effect is a big design feature of the theme, but it's still an expensive operation. Background-positioning a [1920x1080px image](http://akveo.com/blur-admin/assets/img/blur-bg-blurred.jpg) on 11 panels still produces visible lag on high-end hardware, even after some [GPU optimization](https://github.com/akveo/blur-admin/issues/2).

You can get a very close representation of the blur effect by removing the background image and tweaking the background color:

```css
body.badmin-transparent .panel.panel-blur, body.badmin-transparent .panel.panel-blur:hover {
    border-color: rgba(0,0,0,.5);
    background-image:none;
    background-color: rgba(243, 243, 243, 0.15);
}
```

The resulting effect is much faster scrolling and lag reduction, but the background diagonal shader lines are more noticeable through the panels (not very desirable). This compromise might be enough to satisfy some users' needs. Here are some screenshots for comparison - left side is the tweaked version, right side is the original:

![page top](http://i.imgur.com/x0LeXVJ.png)

![halfway down](http://i.imgur.com/tES9KCu.png)

I'm sure you folks have already considered this, but judging by the number of people complaining about scroll lag on your [blog post](http://akveo.com/blog/why-i-hate-template-markets/) it might be worth making a comment about this somewhere.

Thanks for the awesome theme and your hard work!

edit: I see that you have done exactly this for mobile devices ;)

**Comment by tanahara** [comment]
Great fix, I will definitely use this.

**Comment by lugovsky** [comment]
Fixed in v1.2.0
